std/root.ln

```diff
@@ -791,7 +1051,7 @@ export fn prune(n: Node<any>): Tree<any> {
   // This is, unfortunately for now, a sequential algorithm. Hope to figure out a parallel version
   let nodeStack = new Array<int64> [ n.id ];
   let rmdIds = new Array<int64> [ ];
-  seqdo(newseq(pow(2, 62)), fn (): bool {
+  seqdo(newseq(pow(2, 62).getOr(0)), fn (): bool {
```

Comment:
Similarly here, constructing a sequence type shouldn't fail, so this unwrapping is necessary.